docs/tutorials/comparison.md

```diff
@@ -315,7 +319,7 @@ clip3 = core.resize.Lanczos(clip3, format=vs.YUV444P16, range=1)
 
 #### Gamma
 
-Adjusts the gamma level of the video. *This should only be used to fix the QuickTime gamma bug or similar where one source will appear much brighter than the rest.*
+Adjusts the gamma level of the video. *This should only be used to fix QuickTime gamma bugs or similar issues where one source will appear much brighter than the rest.*
```

Comment:
Should just be bug singular, it's a known issue with QuickTime and the result is always the same
https://www.cined.com/quicktime-gamma-shift-bug-what-is-it-and-how-to-combat-it/